scripts/start_radar.sh

```diff
@@ -1,23 +1,28 @@
 #!/bin/bash
-source "${HOME}/.profile"
+source "/home/radar/.profile"
```

Comment:
Why not use the `Home` environment variable? It should always expand out to the users home directory anyways.

Reply:
If a user other than `radar` ran this script, $HOME would expand to a different location. It is unlikely that this script would be ran by anyone other than `radar` however.